Enigmatry.Entry.SmartEnums.Swagger/SmartEnumSwaggerSchemaProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Reflection;
+using Ardalis.SmartEnum;
+using NJsonSchema;
+using NJsonSchema.Generation;
+
+namespace Enigmatry.Entry.SmartEnums.Swagger;
+
+internal class SmartEnumSwaggerSchemaProcessor : ISchemaProcessor
+{
+    public void Process(SchemaProcessorContext context)
+    {
+        var contextualType = context.ContextualType;
+        var schema = context.Schema;
+        var type = contextualType.OriginalType;
+
+        if (type.IsDerivedFromSmartEnum())
```

Comment:
This changes Swagger schema for the smart enums. This needs to be in sync with the Json serializer